src/Compatibility/Microsoft.DotNet.Compatibility/PackageValidationLogger.cs

```diff
@@ -4,9 +4,10 @@
 using System.IO;
 using Microsoft.Build.Framework;
 using Microsoft.DotNet.Compatibility.ErrorSuppression;
+using Microsoft.DotNet.PackageValidation;
 using Microsoft.NET.Build.Tasks;
 
-namespace Microsoft.DotNet.PackageValidation
+namespace Microsoft.DotNet.Compatibility
 {
     internal class PackageValidationLogger : IPackageLogger
```

Comment:
Maybe we should rename this to not mention `Package`?, Maybe, `ValidationLogger`? I would expect this to end up being used by `API Compat` task.